docs/quickstart/network-runner.md

```diff
@@ -12,38 +18,525 @@ That repository's README details the tool.
+curl -X POST -k http://localhost:8081/v1/ping -d ''
+
+# or
+avalanche-network-runner ping \
+--log-level debug \
+--endpoint="0.0.0.0:8080"
+```
+
+#### start a new Avalanche network with five nodes (a cluster)
+
+```bash
+# replace execPath with the path to AvalancheGo on your machine
+# e.g., ${HOME}/go/src/github.com/ava-labs/avalanchego/build/avalanchego
+AVALANCHEGO_EXEC_PATH="avalanchego"
+
+curl -X POST -k http://localhost:8081/v1/control/start -d '{"execPath":"'${AVALANCHEGO_EXEC_PATH}'","numNodes":5,"logLevel":"INFO"}'
+
+# or
+avalanche-network-runner control start \
+--log-level debug \
+--endpoint="0.0.0.0:8080" \
+--number-of-nodes=5 \
+--avalanchego-path ${AVALANCHEGO_EXEC_PATH}
+```
+
+Additional optional parameters which can be passed to the start command:
+
+```bash
+  --plugin-dir ${AVALANCHEGO_PLUGIN_PATH} \
+	--custom-vms '{"subnetevm":"/tmp/subnet-evm.genesis.json"}'
+	--global-node-config '{"index-enabled":false, "api-admin-enabled":true,"network-peer-list-gossip-frequency":"300ms"}'
+	--custom-node-configs" '{"node1":{"log-level":"debug","api-admin-enabled":false},"node2":{...},...}'
+```
+
+`--plugin-dir` and `--custom-vms` are parameters relevant to subnet operation.
+See the [subnet](#network-runner-rpc-server-subnet-evm-example) section for details about how to run subnets.
+
+The network-runner supports avalanchego node configuration at different levels.
+1. If neither `--global-node-config` nor `--custom-node-configs` is supplied, all nodes get a standard set of config options. Currently this set contains:
+    ```bash
+        {
+        "network-peer-list-gossip-frequency":"250ms",
+        "network-max-reconnect-delay":"1s",
+        "public-ip":"127.0.0.1",
+        "health-check-frequency":"2s",
+        "api-admin-enabled":true,
+        "api-ipcs-enabled":true,
+        "index-enabled":true
+        }
+    ```
+2. `--global-node-config` is a JSON string representing a *single* avalanchego config, which will be applied to **all nodes**. This makes it easy to define common properties to all nodes. Whatever is set here will be *combined* with the standard set above.
+3. `--custom-node-configs` is a map of JSON strings representing the *complete* network with individual configs. This allows to configure each node independently. If set, `--number-of-nodes` will be **ignored** to avoid conflicts.
+4. The configs can be combined and will be merged, i.e. one could set global `--global-node-config` entries applied to each node, and also set `--custom-node-configs` for additional entries.
+5. Common `--custom-node-configs` entries override `--global-node-config` entries which override the standard set.
+6. The following entries will be **ignored in all cases** because the network-runner needs to set them internally to function properly:
+    ```bash
+      --log-dir
+      --db-dir
+      --http-port
+      --staking-port
+      --public-ipc
+    ```
+
+
+#### wait for all the nodes in the cluster to become healthy
```

Comment:
What are the response to indicate that the cluster is healthy or not?

Reply:
Great point! The response is actually huge, but contains `healthy:true` at the very end. Will add accordingly.